### PR TITLE
Add Mac notification functions based on terminal notifier

### PR DESCRIPTION
**Pull request by crisidev**
## Changes
* Add Mac notification system using terminal notifier
* Fix typos where Desactivate should be Deactivate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ KeepEye is a vim plugin that invites you to take a break after a huge coding ses
  - *05/25* - add User hl-group support (thanks to [@kristijanhusak](https://github.com/kristijanhusak))
  - *05/25* - add [vim-airline](https://github.com/vim-airline/vim-airline) integration
  - *05/24* - add bell sound feature :bell:
- - *05/23* - add system notification feature (thanks to [@stsewd](https://github.com/stsewd))
+ - *05/23* - add system notification feature for Linux (thanks to [@stsewd](https://github.com/stsewd))
+ - *05/30* - add system notification feature for Mac based on [terminal-notifier](https://github.com/julienXX/terminal-notifier) (thanks to [@crisidev](https://github.com/crisidev))
```
Comment (stsewd) [inline README.md]: I think the more recent changes goes first int this list

Comment (crisidev) [reply README.md]: You are right! Fixing it.

File: autoload/keepeye/feature/bell.vim [hunk 2]
```diff
@@ -9,7 +9,7 @@ function! keepeye#feature#bell#Activate()
   call system(l:cmd)
 endfunction
 
-function! keepeye#feature#bell#Desactivate()
+function! keepeye#feature#bell#Deactivate()
```
Comment (soywod) [inline autoload/keepeye/feature/bell.vim:12]: I'm so ashamed ... thank you to have fixed this English mistake !

**Review by soywod — APPROVED** [review]
Thank you a lot for your contribution :+1:

**Comment by soywod** [comment]
PR merged :tada:

**Pull request merged**